wormhole-connect/src/views/v2/Bridge/AmountInput/index.tsx

```diff
@@ -210,19 +210,12 @@ const AmountInput = (props: Props) => {
                 e.currentTarget.blur();
               },
               step: '0.1',
+              pattern: '[0-9]+([.|,][0-9]{1,2})?',
             }}
             placeholder="0"
             variant="standard"
             value={amount}
             onChange={handleChange}
-            onWheel={(e) => {
```

Comment:
Does removing this prop essentially disable the wheel?

Reply:
I moved it to input props with a bit cleaner implementation couple lines above, this got re added via merge conflict artifacts, so removing again.